style.css

```diff
@@ -0,0 +1,375 @@
+/* Reset  */
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+  margin-top: 5px;
+}
+
+body {
+  min-height: 100vh;
+  margin-left: 40px;
+  margin-right: 40px;
+  scroll-behavior: smooth;
+}
+
+/* Header Section  */
+
+.header-grid {
+  display: grid;
+  grid-template:
+    "twitter-logo home-grid search" auto /
+    auto 1fr auto;
+  gap: 5px;
+}
+
+.twitter-logo {
+  grid-area: twitter-logo;
+  padding-right: 100px;
+}
+
+.home-section {
+  grid-area: home-grid;
+}
+
+.home-grid {
+  display: grid;
+  grid-template-columns: auto auto;
+}
+
+.home-header {
+  justify-content: left;
+  font-size: x-large;
+  font-weight: bolder;
+  padding-top: 5px;
+  padding-left: 50px;
+}
+
+.stars-icon {
+  text-align: right;
+  margin-right: 50px;
+}
+
+.search {
+  grid-area: search;
+  padding-right: 100px;
+  border: 2px solid rgb(226, 225, 225);
+  background-color: rgb(226, 225, 225);
+  border-radius: 30%;
+  text-align: center;
+  text-decoration: black;
+  justify-content: center;
+  padding-right: 250px;
+}
+
+.search-grid {
+  display: grid;
+  grid-auto-flow: column;
+  gap: 5px;
+}
+
+.search-image {
+  margin-top: 7px;
+}
+
+.Search-twitter {
+  padding-top: 15px;
+  padding-left: 15px;
+}
+
+.space1 {
+  width: 50px;
+}
+
+/* Main Section  */
+
+.main-section-grid {
+  display: grid;
+  grid-template:
+    "left-menu middle-main right-column" auto /
+    auto 1fr auto;
+}
+
+.left-main-grid {
+  display: grid;
+  grid-auto-rows: auto;
+}
+
+.left-main-grid > div {
+  display: grid;
+  grid-auto-flow: column;
+  font-weight: bolder;
+}
+
+.left-menu {
+  grid-area: left-menu;
+}
+
+.home-button {
+  text-align: center;
```

Comment:
Some attributes are repeated often. Maybe you could create a "utility class" containing that attribute and apply to any element you want instead.